packages/typespec-test/test/anomalyDetector/generated/openapi/v1.1/openapi.json

```diff
@@ -615,8 +615,7 @@
           "description": "Error message for the current time stamp.",
           "items": {
             "$ref": "#/definitions/Multivariate.ErrorResponse"
-          },
-          "x-ms-identifiers": []
```

Comment:
we don't need to care this i think.

Reply:
refer to https://github.com/Azure/typespec-azure/pull/2607/files